devxicans/platform
# feat: home page localization

## 🔥 Localization

### The overall purpose of this PR is to be able to make our app multilingual and also get the users' preferred language and display the language in the app.

### How to test?
You can run pnpm run dev --filter@1xdev/homepage and see how the path now have the /lang of the users' preferred language, also you can modify the /lang directly on the path, actually we're gonna be supporting **[ en , es ]**.

### Screenshots

**path /en**

![Screenshot 2024-12-18 200024](https://github.com/user-attachments/assets/cfcee75f-386d-4fb2-a93f-628185fa6930)


**path /es**

![image](https://github.com/user-attachments/assets/95570f03-a16a-4b38-895a-3f706d3e3d03)

#### Before

#### After
